### PR TITLE
Managers page authorisation

### PR DESCRIPTION
**Pull request by bodnara**
Related task: # 6813

* added authorisation in managers page for:
* selected roles and their 'mode' based on Perun rules
* add/remove buttons
* missing sidemenu and overview authorisation

### CODE REVIEW

**Review by xkureck — CHANGES_REQUESTED** [review]
image
Please add authorization to resource overview(advanced settings)
i got roles resource admin + resource-self service(and didnt saw anything in resource-settings-overview), then I figured it out that just the role resource admin cannot see it
image


resource-self service shouldnt see the managers page at all because it doesnt have the rights to call the method
![image](https://user-images.githubusercontent.com/32871167/101472970-8e6f8080-3949-11eb-9c9d-4747c4bb95ab.png

**Review by xkureck — CHANGES_REQUESTED** [review]
image
I have resource observer role - i can see in the resource overview advanced settings and then managers as I should but i cant see that in sidemenu which is bug
Wait I just realized that I cant see the advanced settings in the resource section in sidemenu even if I am perun admin
Also please look on the group advaced settings - both sidemenu and overview when you are only member of that group 
image

**Review by xkostka2 — CHANGES_REQUESTED** [review]
image

With role trusted facility admin I get a privileged exception when clicking on advanced settings.

**Comment by bodnara** [comment]
With only role trusted facility admin you shouldn't see advanced settings.

image

I recently pushed a new version so try to update and see if the issue persists.

**Review by xkostka2 — APPROVED** [review]

File: apps/admin-gui/src/app/shared/side-menu/side-menu-item.service.ts [hunk 1]
```diff
@@ -278,19 +278,22 @@ export class SideMenuItemService {
...
+    if (this.authResolver.isManagerPagePrivileged(resource)){
+      links.push({
+        label: 'MENU_ITEMS.RESOURCE.SETTINGS',
+        url: [baseUrl, `settings`],
+        activatedRegex: `${regexStart}/\\d+/resources/\\d+/settings$`,
+        children: [
+          {
+            label: 'MENU_ITEMS.RESOURCE.MANAGERS',
+            url: [baseUrl, `settings`, `managers`],
+            activatedRegex: `${regexStart}\\d+/resources/\\d+/settings/managers$`
```
Comment (xkureck) [inline apps/admin-gui/src/app/shared/side-menu/side-menu-item.service.ts:291]:
```suggestion
            activatedRegex: `${regexStart}/\\d+/resources/\\d+/settings/managers$`
```

**Review by xkureck — CHANGES_REQUESTED** [review]

**Pull request merged**